pkg/operator/ceph/csi/spec.go

```diff
@@ -105,12 +107,13 @@ var (
 // manually challenging.
 var (
 	// image names
-	DefaultCSIPluginImage   = "quay.io/cephcsi/cephcsi:v3.2.1"
-	DefaultRegistrarImage   = "k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.0.1"
-	DefaultProvisionerImage = "k8s.gcr.io/sig-storage/csi-provisioner:v2.0.4"
-	DefaultAttacherImage    = "k8s.gcr.io/sig-storage/csi-attacher:v3.0.2"
-	DefaultSnapshotterImage = "k8s.gcr.io/sig-storage/csi-snapshotter:v3.0.2"
-	DefaultResizerImage     = "k8s.gcr.io/sig-storage/csi-resizer:v1.0.1"
+	DefaultCSIPluginImage         = "quay.io/cephcsi/cephcsi:v3.2.1"
+	DefaultRegistrarImage         = "k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.0.1"
+	DefaultProvisionerImage       = "k8s.gcr.io/sig-storage/csi-provisioner:v2.0.4"
+	DefaultAttacherImage          = "k8s.gcr.io/sig-storage/csi-attacher:v3.0.2"
+	DefaultSnapshotterImage       = "k8s.gcr.io/sig-storage/csi-snapshotter:v3.0.2"
+	DefaultResizerImage           = "k8s.gcr.io/sig-storage/csi-resizer:v1.0.1"
+	DefaultVolumeReplicationImage = "quay.io/csiaddons/volumereplication-operator:latest"
```

Comment:
Still concerned about the `latest` tag. We should use a version tag instead of a floating tag.

Reply:
There are no other tags presently.

Reply:
We own the image, right? Let's ask for versioned tags, even if v0.0.1 to get started.

Reply:
`v0.1.0` image is available `quay.io/csiaddons/volumereplication-operator:v0.1.0`